frontend/src/libs/components/card/card.tsx

```diff
@@ -44,7 +48,12 @@ const Card: React.FC<Properties> = ({
           )}
           {hasIcon && (
             <div className={styles['icon-background']}>
-              <Icon name={iconName as IconName} color={iconColor} />
+              <Icon
+                name={iconName as IconName}
+                color={iconColor as ValueOf<typeof IconColor>}
+                width={iconWidth as number}
+                height={iconHeight as number}
```

Comment:
I setted `Icon` component props as that can be `undefined`, while you made these assertions.
@v0ldemar01, @fshabanov What will be the better approach to handle this issue?